app/views/govuk_publishing_components/components/_chat_entry.html.erb

```diff
@@ -0,0 +1,47 @@
+<%
+  add_gem_component_stylesheet("chat-entry")
+  shared_helper = GovukPublishingComponents::Presenters::SharedHelper.new(local_assigns)
+  href ||= nil
+  heading_text ||= nil
```

Comment:
Do we want the `href`, `heading_text` and `description_text` to be parameters?
My understanding is that they will be same everywhere so it would be much simpler to bake them into the component and avoid the need to update them on every page if they change.
Alternatively we could use the current text as the default and allow it to be overridden.

Reply:
Good thinking. I thought that component options might add a bit more flexibility but perhaps that's not really needed. Default text might be a good idea. @JamesCGDS what do you think?

Reply:
Thanks both. I've made the changes in a separate commit. I've added default text for href, heading, and description. The description text has been updated to the latest version. I've also revised the tests and removed the href condition, as I think we'll always need the href. The only thing I'm still unsure about is what the final href will be, so I'll need to confirm that.

Reply:
Cool, that all sounds good to me 👍